src/main/java/graphql/schema/idl/SchemaTypeChecker.java

```diff
@@ -461,6 +461,52 @@ private void checkInterfacesAreImplemented(List<GraphQLError> errors, TypeDefini
         };
     }
 
+    @SuppressWarnings("SimplifiableIfStatement")
+    private boolean isTypeSubTypeOf(TypeDefinitionRegistry registry, Type maybeSubType, Type superType) {
```

Comment:
this method is complex enough to justify maybe a test just for that.

Reply:
done